src/blocks/DetailsBlock/DetailsBlock.astro

```diff
@@ -0,0 +1,45 @@
+---
+import type { DetailsBlockFragment } from '@lib/types/datocms';
+
+interface Props {
+  block: DetailsBlockFragment
+}
+
+const { block } = Astro.props;
+const splitChar = '|';
+const hasMultipleValues = (value: string) => value.includes(splitChar);
+const splitValues = (value: string) => value.split(splitChar);
+---
+
+<dl class:list={['details-block', `details-block--${block.display}`]}>
```

Comment:
Styles in Astro are scoped. So you don't need to introduce classes here. We want to avoid adding a CSS strategy and leave that choice to the developers using Head Start. Have a look at how this is done in other components.

Reply:
Yeah I know. I did it that way because I thought we were consciously not styling on attributes.